### PR TITLE
Add working group notes for 6/25/2021

### PR DESCRIPTION
**Pull request by arjundeopujari**
### Pull Request Overview

Add working group notes for 6/25/2021

### Testing Strategy

N/A

### TODO or Help Wanted

Please let me know if I misrepresented anyone.  Thanks.

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x] Ran `make prepush`.

### CODE REVIEW

File: doc/wg/core/notes/core-notes-2021-06-25.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Tock Core Notes 2021-06-25
+
+Attending:
+- Alexandru Radovici
+- Arjun Deopujari
+- Brad Campbell
+- Branden Ghena
+- Hudson Ayers
+- Johnathan Van Why
```
Comment (jrvanwhy) [inline doc/wg/core/notes/core-notes-2021-06-25.md:9]: I was unable to attend this call.

File: doc/wg/core/notes/core-notes-2021-06-25.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+- Anthony Quiroga
+
+## Updates
+
+- Pat: I want to introduce Anthony.  He is going to be working over the summer on the CI work that was
+  getting going.  You can expect a PR of how it is going to work later today after I read over it.
+- Leon: We have been getting inquiries on releasing a new "Tock-registers" version.  Unfortunately, the latest
+  Rust nightly build does break the current released version.  We should look into that.  One PR I released has some discussion
+  and we should discuss how to continue from there and we can release a new version accordingly.
+- Johnathan: If they are asking for an updated Tock-registers version, can't we just update Tock to that version?  This might
```
Comment (jrvanwhy) [inline doc/wg/core/notes/core-notes-2021-06-25.md:23]: I don't know who said this, but it wasn't me.

Comment (arjundeopujari) [reply doc/wg/core/notes/core-notes-2021-06-25.md:23]: @jrvanwhy  Sorry for that.  I got your voice mixed up with Branden's.

**Review by jrvanwhy — APPROVED** [review]

**Review by hudson-ayers — APPROVED** [review]

File: doc/wg/core/notes/core-notes-2021-06-25.md (hunk 2 above)
Comment (brghena) [reply doc/wg/core/notes/core-notes-2021-06-25.md]: I'm taking this as a huge compliment!

**Pull request merged**